### PR TITLE
Add tests for all sms views

### PR DESCRIPTION
**Pull request by Awinja-j**
**This pull request introduces the following tests:**

```sms_api_clinical_officer.py```
```sms_api_doctors.py```
```sms_api_health_facilities.py```
```sms_api_nhif_accredited_hospitals.py```
```sms_api_nurses.py```

**Description of Task to be completed:**

- Write tests for doctors, health facilities and NHIF accredited hospitals

**Background context:**

Currently, there are only tests for the nurses' response and not the other views.

**The relevant pivotal tracker story:**

[API] Add tests for all sms api views

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -87,3 +87,8 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+#.DS_Store
+.DS_Store
+/.DS_Store
+/./.DS_Store
```
Comment (DavidLemayian) [inline .gitignore]: There should be a smarter way to ignore this.. `*.DS_Store`?

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
Just one change on this.

I'm hopping you're not using a real phone number here.. In fact for Analytics, let's use a fake one like just 0? Or a combination of 10 digits that's definitely fake if in the future we want to create a validator for this.

File: .gitignore (hunk 1 above)
Comment (Awinja-j) [reply .gitignore]: ah yes. Thanks.

Comment (Awinja-Andela) [reply .gitignore]: Its a fake number. The number is not real.

Comment (Awinja-j) [reply .gitignore]: i have changed the number to '09'

**Comment by DavidLemayian** [comment]
@Awinja-Andela Please resolve conflicts.

**Pull request merged**